src/views/Nft/market/Profile/components/EditProfileModal/StartView.tsx

```diff
@@ -43,7 +43,7 @@ const AvatarWrapper = styled.div`
 const StartPage: React.FC<StartPageProps> = ({ goToApprove, goToChange, goToRemove, onDismiss }) => {
   const { t } = useTranslation()
   const { account } = useWeb3React()
-  const cakeContract = useCake()
+  const cakeContract = useCake(false)
```

Comment:
<img width="742" alt="Screen Shot 2022-02-15 at 13 20 19" src="https://user-images.githubusercontent.com/97418926/154004013-bb7f4a76-57e2-422c-8cba-abce24001585.png">

Could you use `requiresApproval` for this case ?

Reply:
Done